_posts/2016/2016-04-20-jenadevs-docker-party.md

```diff
@@ -0,0 +1,72 @@
+
+Then the Docker Party was launched and we celebrated the [3rd birthday of Docker](https://www.docker.com/community/docker-birthday-3)!
+
+The agenda included a one hour talk along with a demo to introduce the technological foundation as well as different workshops for every skill level:
+
+  1. **Beginners:** The official Docker birthday app tutorial
+  1. **Advanced:** Building your own Dockerfile according to best practises in conjunction with integration tests 
+  1. **Special:** CoresOS Cluster Workshop
+  1. **Hidden - also known as exploding kittens:** Orchestration Workshop by [Jérôme Petazzoni](https://twitter.com/jpetazzo), the creator of [Docker-in-Docker](https://github.com/jpetazzo/dind) 
+
+## Introduction talk with demo and best practises
+
+In the first place the talk focused on the Docker basics, the components of the Docker ecosystem and the general tooling workflow of building images, running containers and pushing them to the Docker Hub.
+We had a demo on the basic commands along with some live coding on the terminal, where several containers were also inspected at runtime.
+The second part considered some best practises for writing your own Dockerfile, which was based on the [official Docker recommendations](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices), tips by [Michael](http://crosbymichael.com/dockerfile-best-practices.html) [Crosby](http://crosbymichael.com/dockerfile-best-practices-take-2.html) as well as the experiences of [epagesdevs](http://twitter.com/epagesdevs) with developing, testing, integrating, deploying and running supportive images for our eCommerce platform.
+This was a nice opportunity to also show some of our typical use cases at ePages, e.g. in the Continuous Delivery Pipeline, for integration testingor in the operations area of our own and provider infrastructure based already partly on [Kubeneretes](http://kubernetes.io).
+The talk finished with some typical base images as well as a discussion about official app environment images.
+
+## Containers, Containers, Con... er Workshops!
+
+Upfront, all necessary material and code for the workshops was hosted on the jenadevs [GitHub](https://github.com/jenadevs) account, which even offers the [complete presentation](https://github.com/jenadevs/jenadevs-meetup-001-docker-party).
+Great news: _All material of **[jenadevs](https://github.com/jenadevs)** is free and will always be open-source!_
+
+### Beginners Workshop
+
+The goal of the beginner workshop was to setup the Docker environment, learn the basic commands, run the workflow and become familiar with Docker Compose.
+The participants followed the [official Docker birthday tutorial](https://github.com/docker/docker-birthday-3/blob/master/tutorial.md), while the more experienced Docker users, Benjamin Nothdurft and Kay Abendroth, offered assistance.
```

Comment:
... while more experienced Docker users offered assistance.

Please remove the names.